glpk/meta.yaml

```diff
@@ -17,6 +17,16 @@ requirements:
   build:
     - gmp      [not win]
 
+test:
```

Comment:
Ugh, we have some issues with test-only dependencies.  If you don't mind, please add it as a runtime dependency also, so that it gets installed correctly.  I don't want our test-dependency issue to hold up your PR.  Add a comment about this to the recipe, if you'd like.

Ping @kalefranz @mcg1969 - if either of you run into code that might address this problem, let me know.